DEVELOPER.md

```diff
@@ -0,0 +1,7 @@
+# Useful tips
+
+## How to make protobuf imports work in your IDE
+1. `go list -f '{{ .Dir }}' -m github.com/gogo/protobuf`
+2. Add the directory from step 1 to your protobuf imports in the plugin you use
```

Comment:
This will work as long as no one will update the version we are using. Then you either use old cached version and might get subtle errors or it stops working if the cache is cleaned.

Reply:
Add a note to redo this if gogo/protobuf is updated in go.sum

Since this is just IDE auto-complete, I am not too worried about false error messages, just mention to redo it if IDE acts funky.

Reply:
good catch